### PR TITLE
upgrade kafka client to 2.0.0

### PR DESCRIPTION
**Pull request by jsvd**
solves #289

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,5 +14,7 @@ matrix:
     env: LOGSTASH_BRANCH=5.6
   fast_finish: true
 install: true
-script: ci/build.sh
+before_script: ci/build.sh
+script: ci/run.sh
+script: ci/cleanup.sh
```
Comment (robbavey) [inline .travis.yml]: Shouldn't this be `after_script`?

**Comment by MarcusCaepio** [comment]
When will this be merged? 
It currently looks like input plugin kafka is not usable with kafka 2 at logstash 6.4

**Comment by cricket007** [comment]
Note: This drops Java 7 support. Should that should be documented somewhere?

**Comment by allenmchan** [comment]
I believe logstash also officially doesn’t support java 7. Probably not a good idea to be on Java 7.

**Comment by cricket007** [comment]
Yeah, I haven't been following ELK products closely enough to know the JVM support levels, but does seem latest installation docs say required Java8, so nevermind! 👍

**Comment by jsvd** [comment]
@robbavey ready for review again 👍

**Comment by jsvd** [comment]
nice catch, that file should actually be deleted from git as it's auto generated during `rake vendor`, but I'll leave that out of this PR

**Pull request merged**